kubernetes/config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: issecrets
```

Comment:
This is one secret, not multiple. The kubernetes secret could be named `issue-sync`